tests/phpunit/tests/link/postNavigationLinkAttributeFilters.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * Test post navigation link attribute filters.
+ *
+ * @since 6.2.0
+ * @ticket 55751
+ *
+ * @group link
+ *
+ * @covers ::next_posts_link_attributes
+ * @covers ::previous_posts_link_attributes
+ */
+class Tests_Link_PostNavigationLinkAttributeFilters extends WP_UnitTestCase {
+
+	/**
+	 * Create posts for the tests.
+	 *
+	 * @param WP_UnitTest_Factory $factory
+	 */
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		$factory->post->create_many( 3 );
+		global $wp_query, $paged;
```

Comment:
```suggestion
		global $wp_query, $paged;

		$factory->post->create_many( 3 );
```
`global` typically comes first in a function/method.